src/Picqer/Financials/Exact/ItemAssortment.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Picqer\Financials\Exact;
+
+/**
+ * Class ItemAssortment
+ *
+ * @package Picqer\Financials\Exact
+ * @see https://start.exactonline.nl/docs/HlpRestAPIResourcesDetails.aspx?name=LogisticsItemAssortment
+ *
+ * @property string $ID ID of ItemAssortment
+ * @property int $Code Code of ItemAssortment
+ * @property string $Description Description of ItemAssortment
+ * @property int $Division Division
+ * @property properties $Properties Properties of this ItemAssortment
```

Comment:
Guess the code generation doesn't support "properties" mapping. Would this be off the type array?

Reply:
If you could indeed check and fix this and add the unit tests, I can merge the PR.